tutorials/create-cloud-build-image-factory-using-packer/windows/scripts/disable-uac.ps1

```diff
@@ -0,0 +1,4 @@
+Write-Output "+++ Disabling UAC… +++"
```

Comment:
Why is disabling UAC required?

Reply:
This is required to invoke remote shells for the bootstrap/chocolatey scripts. If PS commands are passed directly as part of the packer provisioner stage, you don't need to bypass UAC, but I thought it'd be best for readability and maintenance if separated by scripts for various stages. See below on UAC from packer site.

Source: https://www.packer.io/intro/getting-started/build-image/
..
# Configure UAC to allow privilege elevation in remote shells
..